packages/expo/src/executors/export/schema.json

```diff
@@ -40,7 +41,8 @@
     "bundler": {
       "enum": ["metro", "webpack"],
       "description": "Choose the bundler to compile for",
-      "default": "metro"
+      "default": "metro",
+      "x-priority": "important"
```

Comment:
i'm not sure if this should be important... it has a default and probably folks will use the same bundler throughout the app. So it will be set in `nx.json` and not set again and again.

Reply:
removed